perun-ldapc/src/main/java/cz/metacentrum/perun/ldapc/processor/impl/EventProcessorImpl.java

```diff
@@ -94,18 +95,83 @@
 	//UserExtSources patterns
-	private Pattern addUserExtSource = Pattern.compile("UserExtSource:\\[(.*)\\] added to User:\\[(.*)\\]");
-	private Pattern removeUserExtSource = Pattern.compile("UserExtSource:\\[(.*)\\] removed from User:\\[(.*)\\]");
+	private Pattern addUserExtSourcePattern = Pattern.compile("UserExtSource:\\[(.*)\\] added to User:\\[(.*)\\]");
+	private Pattern removeUserExtSourcePattern = Pattern.compile("UserExtSource:\\[(.*)\\] removed from User:\\[(.*)\\]");
 
 	//CONSTANTS
 	private static final String LDAP_NAME = "ldap";
 	private static final String PATH_TO_ERROR_FILE = "./errorParseMessageForLDAP.log";
 	private static final String PATH_TO_NOT_EXECUTED_ERROR_FILE = "./notExecutedMessages.log";
 	private static final DateFormat DATE_FORMAT = new SimpleDateFormat("yyyy/MM/dd HH:mm:ss");
 
+	//PERUN ATTRIBUTES NAMES
+	public static final String perunAttrPreferredMail = "preferredMail";
+	public static final String perunAttrMail = "mail";
+	public static final String perunAttrOrganization = "organization";
+	public static final String perunAttrPhone = "phone";
+	public static final String perunAttrUserCertDNs = "userCertDNs";
+	public static final String perunAttrBonaFideStatus = "elixirBonaFideStatus";
+	public static final String perunAttrSchacHomeOrganizations = "schacHomeOrganizations";
+	public static final String perunAttrEduPersonScopedAffiliations = "eduPersonScopedAffiliations";
+	public static final String perunAttrLibraryIDs = "libraryIDs";
+	public static final String perunAttrEntityID = "entityID";
+
+	//LDAP ATTRIBUTES NAMES
+	public static final String ldapAttrAssignedToResourceId = "assignedToResourceId";
+	public static final String ldapAttrAssignedGroupId = "assignedGroupId";
+	public static final String ldapAttrDescription = "description";
+	public static final String ldapAttrCommonName = "cn";
+	public static final String ldapAttrPerunUniqueGroupName= "perunUniqueGroupName";
+	public static final String ldapAttrEduPersonPrincipalNames = "eduPersonPrincipalNames";
+	public static final String ldapAttrPreferredMail = perunAttrPreferredMail;
+	public static final String ldapAttrMail = perunAttrMail;
+	public static final String ldapAttrOrganization = "o";
+	public static final String ldapAttrTelephoneNumber = "telephoneNumber";
+	public static final String ldapAttrUserCertDNs = "userCertificateSubject";
+	public static final String ldapAttrBonaFideStatus = "bonaFideStatus";
+	public static final String ldapAttrSchacHomeOrganizations = perunAttrSchacHomeOrganizations;
+	public static final String ldapAttrEduPersonScopedAffiliations = perunAttrEduPersonScopedAffiliations;
+	public static final String ldapAttrLibraryIDs = perunAttrLibraryIDs;
+	public static final String ldapAttrUidNumber = "uidNumber;x-ns-";
+	public static final String ldapAttrLogin = "login;x-ns";
```

Comment:
This should probably be "login;x-ns-"

Reply:
Nice one, yes this definitely should be "ns-". Thank you. It is fixed now.